doc/guides/west/repo-tool.rst

```diff
@@ -79,6 +79,74 @@ Finally, any repository managed by a west installation can contain
 provided by that project. This includes the manifest repository and any project
 repository.
 
+Topologies supported
```

Comment:
Why did you remove this from the west documentation then put it back?

Can you clean this up in the commits?